src/main/kotlin/uk/gov/justice/digital/hmpps/pecs/jpc/service/reports/ImportReportsService.kt

```diff
@@ -44,9 +44,23 @@ class ImportReportsService(
     failIfDateOfImportNotInPast(date)
     failIfAnyReportFilesAreMissingOn(date)
 
-    importMovesJourneysEventsOn(date)
-    importPeopleOn(date)
-    importProfilesOn(date)
+    val maybeMoves = importMovesJourneysEventsOn(date)
+    val maybePeople = importPeopleOn(date)
+    val maybeProfiles = importProfilesOn(date)
+
+    if (maybeMoves != null && maybePeople != null && maybeProfiles != null) {
+      auditService.create(
```

Comment:
Decided to squash this into 1 audit event as this feels cleaner in the long run.